src/main/java/info/meoblast001/seallyracing/states/PlayState.java

```diff
@@ -63,9 +107,22 @@ public void update(float tpf) {
     followCamera.update();
 
     // Update player character.
-    player.rotate(-PLAYER_TORQUE * tpf, 0.0f, 0.0f);
-    Vector3f forward = player.getLocalRotation().mult(Vector3f.UNIT_Z).
-      mult(-PLAYER_SPEED * tpf);
-    player.move(forward);
+    Vector3f forward = player.getLocalRotation().mult(Vector3f.UNIT_Z);
+    player.move(forward.mult(PLAYER_SPEED * tpf));
```

Comment:
At the current point, yes. There are of course reasons this could change depending on what functionality is added to the game.